src/panel-manager.js

```diff
@@ -7,8 +7,20 @@ genotet.panelManager = {};
 
 /** @const {number} */
 genotet.panelManager.COLLAPSED_WIDTH = 32;
-/** @const {number} */
-genotet.panelManager.TRANSITION_TIME = 300;
+/** @private @const {number} */
+genotet.panelManager.TRANSITION_TIME_ = 300;
+/** @private @const {number} */
+genotet.panelManager.TOGGLE_BTN_HEIGHT_ = 50;
+
+/**
+ * The difference between the width and other attributes of the tabs container.
```

Comment:
If it's not JSDoc you don't need /** ... */ comments. Just // will be fine.

Besides this explanation is still very confusing. I cannot get it at all ?_?...